### PR TITLE
Add backup usecase for home user to adopters.md

### PR DESCRIPTION
**Pull request by kmova**
Found the OpenEBS adoption story for home backup solution over
here: https://twitter.com/MaartjeME/status/1261747549272236032

It will be helpful to share this story to OpenEBS community.

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

**Comment by kmova** [comment]
@meyskens would you be ok to share your use case over here?

also, I wanted to confirm with you on the OpenEBS storage engine you have set up for this use case.

File: adopters/users/meyskens/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+_backup solution for home laptop_
+
+**Stateful Applications that you are running on OpenEBS**
+- Minio
+
+**Type of OpenEBS Storage Engines behind the above application - cStor, Jiva or Local PV**
+- Jiva
```
Comment (meyskens) [inline adopters/users/meyskens/README.md:7]: In this use case I use cStor

File: adopters/users/meyskens/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+**A brief description of the use case or details on how OpenEBS is helping your projects.**
+- Restic uploads my laptop to Minio running on my home k8s cluster (powered by openebs). A “mc mirror” pod then uploads it to the cloud. I get full speed backups on my laptop and still have a cloud copy
```
Comment (meyskens) [inline adopters/users/meyskens/README.md:10]:
```suggestion
- Restic uploads my laptop to Minio running on my home Kubernetes cluster. Minio runs on a 2 replica OpenEBS cStor PV . A`“mc mirror` pod then watches the bucket and uploads it's content to the cloud. I get full speed backups on my laptop and still have a cloud copy```

Expanded a bit more as I do not have a character limit here :)

**Comment by kmova** [comment]
Thank you @meyskens - incorporated the review comments!

**Pull request merged**